DPGAnalysis/HcalTools/plugins/CMTRawAnalyzer.cc

```diff
@@ -8653,7 +8736,7 @@ void CMTRawAnalyzer::fillDigiAmplitude(HBHEDigiCollection::const_iterator& digiI
   double errorBtype = 0.;
 
   //    int TSsize = 10;
-  int TSsize = 10;
+  int TSsize = 10;  //HEHB for Run2
   //     if((*digiItr).size() !=  10) std::cout << "TSsize HBHE != 10 and = " <<(*digiItr).size()<< std::endl;
   if ((*digiItr).size() != TSsize)
```

Comment:
```suggestion
  if ((*digiItr).size() != 10)
    errorBtype = 1.;
  const in TSsize = digiItr->size();
```
... and then the same TSsize can be used in the checks as for example at lines 8880, 8882, 8884, 8886 below

Reply:
This structure with TSsize repeats in quite several other parts in the code: the very same comments apply also in those cases